packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -39,39 +48,23 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>>
       // TODO: determine whether this can be removed
       // eslint-disable-next-line react/no-unused-state
       suggestionItems: this.props.suggestionItems ? (this.props.suggestionItems as T[]) : null,
-      selectedItems: this.props.defaultSelectedItems
-        ? (this.props.defaultSelectedItems as T[])
-        : this.props.selectedItems
-        ? (this.props.selectedItems as T[])
-        : null,
     };
-
-    this.floatingPickerProps = this.props.floatingPickerProps;
-    this.selectedItemsListProps = this.props.selectedItemsListProps;
   }
 
   public get items(): any {
-    return this.state.selectedItems ?? this.selectedItemsList.current?.items ?? null;
+    if (this.props.selectedItems !== null && this.props.selectedItems !== undefined) {
+      return this.props.selectedItems;
+    }
+    if (this.selectedItemsList.current) {
+      return this.selectedItemsList.current.items;
+    }
+    return this.props.defaultSelectedItems;
```

Comment:
Nitpicky thing but you could optionally change this back to the nullish coalescing syntax (it does the same thing as these checks). `this.props.selectedItems ?? this.selectedItemsList.current?.items ?? this.props.defaultSelectedItems`